### ISSUE TITLE
Out of memory issue

### ISSUE THREAD

**Issue by HamedEmine** [paator/eJayDecompressor#9]
Hello, first off, thank you so much for writing this tool!

But I'm having an issue where the PXD file is apparently too big...

```
Version 1.3
Initializing pxd32d5_d4.dll
Converting raw data file techno40a.pxd to wav format...
Out of memory.
```

That file is 1.67GB, is there any way around this?

**Comment by paator** [comment]
Hi @HamedEmine, thanks for opening an issue. I'll look into it :)

**Comment by BKVad1m** [comment]
This is multiPXD can be used eJ_Tool.dll

**Comment by BKVad1m** [comment]
@paator This issue is not planned due to big PXD package using eJ_Tool.dll. Ripped Techno 4 working just fine using eJ_Tool.dll

**Comment by paator** [comment]
Thank you for checking. @HamedEmine can you confirm if eJ_Tool.dll fixes the problem?

**Comment by HamedEmine** [comment]
Hello again, I have eJ_Tool.dll in the same directory, but it doesn't "use/call" it

What do I need to do exactly?

**Comment by paator** [comment]
@Vad1mCSO2 how did you use eJ_Tool.dll for pxd file? This script picks pxd32d5_d4.dll automatically for PXD files.

**Comment by BKVad1m** [comment]
> @Vad1mCSO2 how did you use eJ_Tool.dll for pxd file? This script picks pxd32d5_d4.dll automatically for PXD files.

yes, this ej_tool.dll to rip compressed pxd sound file when file size over than 64MB, pxd32d5_d4.dll to rip single pxd files less than 64MB

**Comment by DanceMore** [comment]
can anyone share at least `md5sum` or `sha1sum` of "known working" DLLs? there's a lot of Versions of E-Jay software released over the years.
